bin/post_compile

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+indent() {
+  sed "s/^/       /"
+}
+
+puts-step() {
+  echo "-----> $@"
+}
+
+puts-step "Running migrations..."
+
+python manage.py db upgrade 2>&1 | indent
```

Comment:
I think you want to add `exit "${PIPESTATUS[0]}"` after this line; `indent` is consume the exit code.